src/Illuminate/Container/BoundMethod.php

```diff
@@ -5,71 +5,78 @@
 use Closure;
 use ReflectionMethod;
 use ReflectionFunction;
+use Illuminate\Support\Arr;
 use InvalidArgumentException;
 
 class BoundMethod
 {
+    /**
+     * @var \Illuminate\Foundation\Application
```

Comment:
This should be `\Illuminate\Contracts\Container\Container`

The `illuminate/container` package can't reference something from the `Foundation` namespace.